.github/workflows/ci.yml

```diff
@@ -49,8 +50,42 @@ jobs:
             - name: PHPStan checks
               run: composer stan
 
-            - name: CodeStyle checks
-              run: composer cs-check
+    cs-check:
+        runs-on: ubuntu-20.04
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version: [
+                    '7.4',
+                    '8.0',
```

Comment:
And in this case 8.1 should be used here instead of 8.0

Reply:
cs-check 8.1 is not compatible with 7.4 
As result, we decided that for the checker we will use only 7.4 and 8.0

Reply:
I reached, it is not reproduced now 
Addedreplaced 8.0 to 8.1